dataland-frontend/tests/component/components/pages/ApiKeyPage.cy.ts

```diff
@@ -18,4 +19,52 @@ describe("Component test for ApiKeyCard.vue", () => {
     cy.get('[data-test="apiKeyInfo"]').find("textarea").should("have.attr", "readonly");
     cy.get('[data-test="apiKeyInfo"]').find("textarea").invoke("val").should("eq", "abcdefghijklmnoprstwxyz123456789");
   });
+
+  it(
+    "If api key does not exist yet, you can open a creation popup, set an expiration date in the dropdown, or " +
+      "close the whole popup again",
```

Comment:
The term popup feels kind of wrong to me here. But I don't have a better proposal right now.

Reply:
Ja hast recht.
Wie wäre es einfach mit:
`If api key does not exist yet, you can begin to create one, set an expiration date in the dropdown, or close the creation again`

Reply:
Schon besser

Reply:
Ok mache ich und sage Bescheid